src/fdosecrets/objects/Collection.cpp

```diff
@@ -490,7 +490,6 @@ namespace FdoSecrets
         // When the group object is normally deleted due to ~Database, the databaseReplaced
         // signal should be first emitted, and we will clean up connection in reloadDatabase,
         // so this handler won't be triggered.
-        QPointer<Database> db = m_backend->database().data();
```

Comment:
This was unused; the lambda was shadowing it with its own `db` which turned out to be the only one needed.

Reply:
Right, never mind. I misread the line.